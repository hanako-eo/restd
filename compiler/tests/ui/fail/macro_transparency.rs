#[compiler::macro_transparency(transparency)]
macro_rules! print {
    () => {}
}

fn main() {}
