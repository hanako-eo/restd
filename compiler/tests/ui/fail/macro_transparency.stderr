error: should be opaque, semiopaque, semitransparent or transparent
 --> tests/ui/fail/macro_transparency.rs:1:32
  |
1 | #[compiler::macro_transparency(transparency)]
  |                                ^^^^^^^^^^^^
